.github/workflows/release.yaml

```diff
@@ -6,6 +6,25 @@ on:
       - "v*"
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+
+      - name: Get changed files
+        id: changed
+        uses: tj-actions/changed-files@v40
```

Comment:
I fear this may cause the same problem we were seeing with the other branches. Any chance we can validate it before merging to main?